### DISCUSSION TITLE
Version 1.24 missing Monitor tab in UI

### DISCUSSION THREAD

**Discussion by pbdger** [jaegertracing/jaeger#3155] [Q&A]
Hi, 

in https://github.com/jaegertracing/jaeger/blob/master/CHANGELOG.md of version 1.24 the new Monitor tab in UI is announced. After update to 1.24 I can't find the new tab.

I can not find any hint in the documentation. Is it really part of version 1.24 ? And how to enable it ?

**Comment by jpkrohling** [comment]
The changelog should be changed: what's listed there is preparation work for the monitor tab. The actual tab will come in a couple of releases.

cc @jkowall, @albertteoh

**Reply by jkowall** [reply]
Yep, we are finishing up some design but it's coming very soon. @yoave23 is the team lead working on it, so maybe he can provide more accurate guidance.

**Reply by yoave23** [reply]
sure, as @jkowall mentioned we're working on finalizing the code and docs prior to creating the PR. 
the feature is mostly ready and has already been tested internally with real data.
ETA for the PR is about 3-4 weeks.
